### ISSUE TITLE
[BUG] No body hair option?

### ISSUE THREAD

**Issue by KingMeric** [qbcore-fivem/qb-clothing#209]
No body hair option in any of the menu, or in the files. Not sure if its supposed to be there or not, seen a topic closed ages ago about this and was wondering what was happening with it.

**Comment by TheRealSago** [comment]
Yes, please add this and the option for a second Haircolour would be awesome!

**Comment by Expl-icit** [comment]
has there ever been a fix to this?
